docs/SUMMARY.md

```diff
@@ -3,63 +3,52 @@
-* Configuring registries
-    * [AWS ECR](guides/registries/aws-ecr.md)
-    * [GCP Artifact Registry](guides/registries/gcp-artifact-registry.md)
-    * [Azure ACR](guides/registries/azure-acr.md)
-    * [Self-signed certificates](guides/registries/self-signed.md)
-
-## 🌍 Earthfiles
 
+## 📖 Docs
+
+* Guides
+    * [Authenticating Git and image registries](guides/auth.md)
+    * [Target, artifact and command referencing](guides/target-ref.md)
+    * [User-defined commands (UDCs)](guides/udc.md)
+    * [Managing cache](guides/cache.md)
+    * [Advanced local caching](guides/advanced-local-caching.md)
+    * [Shared cache](guides/shared-cache.md)
+    * [Using Docker in Earthly](guides/docker-in-earthly.md)
+    * [Cloud secrets](guides/cloud-secrets.md)
+    * [Integration Testing](guides/integration.md)
+    * [Debugging techniques](guides/debugging.md)
+    * [Multi-platform builds](guides/multi-platform.md)
+    * Configuring registries
+        * [AWS ECR](guides/registries/aws-ecr.md)
+        * [GCP Artifact Registry](guides/registries/gcp-artifact-registry.md)
+        * [Azure ACR](guides/registries/azure-acr.md)
+        * [Self-signed certificates](guides/registries/self-signed.md)
 * [Earthfile reference](earthfile/earthfile.md)
-* [Builtin args](earthfile/builtin-args.md)
-* [Excluding patterns](earthfile/earthignore.md)
-
-## ⌨️ The `earthly` command
-
-* [Earthly command reference](earthly-command/earthly-command.md)
-
-## 📄 Configuration
-
+    * [Builtin args](earthfile/builtin-args.md)
+    * [Excluding patterns](earthfile/earthignore.md)
+* [The `earthly` command](earthly-command/earthly-command.md)
 * [Configuration reference](earthly-config/earthly-config.md)
+* [CI integration](./ci-integration.md)
+* Misc
+    * [Alternative installation](./alt-installation.md)
+    * [Definitions](definitions/definitions.md)
+    * [Data collection](data-collection/data-collection.md)
 
 ## 🧙 Examples
 
 * [Overview](examples/examples.md)
-* [Go](examples/go.md)
-* [Java](examples/java.md)
-* [JS](examples/js.md)
-* [C++](examples/cpp.md)
-* [Python](examples/python.md)
-* [Monorepo](examples/monorepo.md)
-* [Multirepo](examples/multirepo.md)
-* [Earthly](examples/earthly.md)
-
-## ⚙️ CI examples
-
-* [Overview](ci-examples/ci-examples.md)
-* [Circle CI](ci-examples/circle-integration.md)
-* [GitHub Actions](ci-examples/gh-actions-integration.md)
-* [AWS CodeBuild](ci-examples/codebuild-integration.md)
-
-## 💼 Misc
-
-* [Definitions](definitions/definitions.md)
-* [Data collection](data-collection/data-collection.md)
+* Per-language Earthfile examples
+    * [Go](examples/go.md)
+    * [Java](examples/java.md)
+    * [JS](examples/js.md)
+    * [C++](examples/cpp.md)
+    * [Python](examples/python.md)
+* Other Earthfile examples
+    * [Monorepo](examples/monorepo.md)
+    * [Multirepo](examples/multirepo.md)
+    * [Earthly](examples/earthly.md)
+* [CI examples](ci-examples/ci-examples.md)
```

Comment:
This point to `docs/ci-examples/ci-examples.md`, which seems to have been deleted in this PR; or am I missing something here?

Reply:
Ah forgot to remove the link. Thanks for finding this.